build.gradle

```diff
@@ -32,11 +32,12 @@ buildscript {
 allprojects {
     apply plugin: "spinnaker.project"
     apply plugin: "groovy"
+    apply plugin: 'java'
```

Comment:
The java plugin is implicit with the groovy plugin.

Reply:
ah good to know, that was a remnant of me trying to debug an issue with jacoco